es-app/src/main.cpp

```diff
@@ -169,9 +168,6 @@ int main(int argc, char* argv[])
 	unsigned int width = 0;
 	unsigned int height = 0;
 
-	std::locale::global(boost::locale::generator().generate(""));
```

Comment:
Sorry... I based these changes off of RecalBox, and it was 3:00 in the morning when finishing these changes. I was tired and I will fix this.